### ISSUE TITLE
Tooltip name property too long

### ISSUE THREAD

**Issue by Angelk90** [pmndrs/leva#241]
Hi @dbismut ,
as you can see in the image below, when there are names with too long properties it is not possible to read the name of the property, it would be useful to have a tooltip that shows the name when passing over the property name.

<img width="276" alt="Schermata 2021-06-11 alle 19 50 37" src="https://user-images.githubusercontent.com/20476002/121729252-8823b200-caee-11eb-8ef7-243e6137f471.png">

**Comment by dbismut** [comment]
You're right, there should be a `title` prop on labels. Note that the `hint` option on the input allows for a longer tooltip message.

**Comment by Angelk90** [comment]
@dbismut : Trying the various examples on the Storybook, I can't find anyone using title.
Does it still need to be implemented?

**Comment by dbismut** [comment]
Yes, this needs a PR. I mean the `title` html prop. https://developer.mozilla.org/fr/docs/Web/HTML/Global_attributes/title

**Comment by Angelk90** [comment]
@dbismut : I understand, it's a pity that graphically you can't get such a result: 
<img width="161" alt="Schermata 2021-06-11 alle 20 16 32" src="https://user-images.githubusercontent.com/20476002/121732087-1483a400-caf2-11eb-85db-ce548d89e546.png">

For example the possibility of being able to use a custom tooltip.

**Comment by dbismut** [comment]
Custom tooltip exists with the `hint` prop. https://leva.pmnd.rs/?path=/story/misc-input-options--hint

**Comment by Angelk90** [comment]
@dbismut : I missed this thing, but not using hint as description, it is not possible to customize it as for example:
- Background color of the tooltip hint
- Text color of the tooltip hint
- Show the tooltip hint in a different position from above the property, for example to the left of it

**Comment by dbismut** [comment]
Colors can be themed. For the rest no, not possible.
